2-projects/1-todo-list/script.js

```diff
@@ -1,25 +1,86 @@
+let list = document.querySelector("#todo-list");
 function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  list.innerHTML = "";
+  /*
+  Write your code to create todo list elements with completed and delete buttons here,
+  all todos should display inside the "todo-list" element.
+  */
+
+  todos.forEach((task) => {
+    const li = document.createElement("li");
```

Comment:
Works and looks nice :) Some nitpicking:

Reverting to non descriptive names here. i_check and i_trash are particaully bad I'm afraid as they also don't follow the same naming format as the rest. Works though, but there is one bug. When you delete some items and add new ones the deleted items magically reappear as they haven't been removed from the list. Function is a bit big as well, ideally break it up into a few functions